lib/circuit.js

```diff
@@ -271,6 +275,10 @@ class CircuitBreaker extends EventEmitter {
     return this[HYSTRIX_STATS];
   }
 
+  get prometheusMetrics () {
```

Comment:
We could probably just call this `metrics()` since if the target changes (e.g. no longer using Prometheus) we don't have to change the name.